groclient/experimental.py

```diff
@@ -0,0 +1,115 @@
+from groclient.client import GroClient
+from groclient import lib
+
+
+class Experimental(GroClient):
+    """
+    Experimental class to consume v2prime data.
+    """
+    # Query parameters other than these are ignored.
+    ALLOWED_SELECTIONS = {
```

Comment:
reading lib.get_data_points() again, I think we don't need to declare all allowed selections here , we should make it flexible for python client. only 5 tuples are required, the others are optional